ariadne/contrib/tracing/opentracing.py

```diff
@@ -65,7 +66,7 @@ def filter_resolver_args(
         if not self._arg_filter:
             return args
```

Comment:
This value also needs to be made safe. `_arg_filter` is user-provided filter for unsafe args (like passwords), but we also log args when no user filter was provided.

Reply:
Fixed.